test/integration-test.sh

```diff
@@ -298,3 +298,65 @@ for i in `seq 1 10`; do
   log "Node 1 send message to node 5"
   send_message "${api1}" "${addr5}" "hello, world" "" 600
 done
+
+# redeem tickets
+log "Node 2 should redeem all tickets"
```

Comment:
I think it should also check that the amount of redeemed tickets changes as a result of this operation. The actual numbers don't need to be specific though, but the change is important.

Reply:
Actually, not sure but am afraid this may lead to flaky tests, since we may redeem tickets but at the same time get a greater or equal amount right before we check `tickets` again.

Reply:
As discussed, opened issue https://github.com/hoprnet/hoprnet/issues/3306 to track this.